apps/src/sites/studio/pages/levels/editors/_all.js

```diff
@@ -54,6 +54,10 @@ function initPage() {
     $("#plusAnswerContainedLevel").prev().clone().insertBefore("#plusAnswerContainedLevel");
   });
 
+  $("#plusPreloadAssetList").on("click", () => {
+    $("#plusPreloadAssetList").prev().clone().insertBefore("#plusPreloadAssetList");
```

Comment:
Nit: I believe this will clone the entire input, including the entered value; something like `$("#plusPreloadAssetList").prev().clone().val("").insertBefore("#plusPreloadAssetList");` will be necessary to give us a new, empty input

Reply:
yes, good point. we're doing this in another spot, but we should probably fix all of them to do what you are suggesting. Next time I'm in there 😄